app/models/solidus_avatax_certified/request/return_tax.rb

```diff
@@ -1,6 +1,7 @@
 module SolidusAvataxCertified
   module Request
     class ReturnTax < SolidusAvataxCertified::Request::Base
+      NO_TAX_EXCEPTIONS = ['Late Delivery', 'Late Shipping']
```

Comment:
I think these should be in our codebase as they are custom to us and we may add more reasons here which shouldn't require gem update